source/features/default-branch-button.tsx

```diff
@@ -19,6 +19,7 @@ async function init(): Promise<false | void> {
 	}
 
 	const url = new GitHubURL(location.href);
+	url.filePath = url.filePath || filePathFromSearch();
 	// The branch selector will be on `isRepoCommitList()` **unless** you're in a folder/file
 	if (pageDetect.isRepoCommitList() && url.filePath.length > 0) {
 		return false;
```

Comment:
I wonder if in this case we should just test the branch selector and drop this condition.

```suggestion
	const branchSelector = await elementReady('#branch-select-menu');
	// The branch selector is missing from History pages of files and folders (it only appears on the root)
	if (!branchSelector) {
		return false;
```